sim/core/spell_result.go

```diff
@@ -457,6 +488,7 @@ func (result *SpellResult) applyTargetModifiers(spell *Spell, attackTable *Attac
 		return
 	}
 
+	// TODO: Add other schools. Multischools should then chose highest.
```

Comment:
Is this still TODO?

Reply:
Yes. Implementing it most likely requires adding the coefficient to all spells, because it would be required at this point for non-physical spells. I'll look into that after this is done.

Reply:
Ok so you think this isn't a blocking issue?